nevergrad/optimization/oneshot.py

```diff
@@ -29,14 +29,20 @@ def __init__(self, instrumentation: Union[int, Instrumentation], budget: Optiona
 
     def _internal_ask(self) -> ArrayLike:
         # pylint: disable=not-callable
+        if self._parameters.quasi_opposite == "quasi" and self._num_ask % 2:
```

Comment:
I'll tend to restructure this:
```python
if self._num_ask % 2:
   if param.quasi_opposite == ...:
        ...
   elif ...:
        ...
```
I tend to find it slightly simpler to read.

Also, if I am correct, with `middle_point == True`, this can lead to a second middle point being asked, which is not optimal.